src/test/e2e/swagger/endpoints.spec.ts

```
@@ -0,0 +1,154 @@
+export type SwaggerLocators = {
+  apiDocumentation: Locator;
+  navigation: Locator;
+  authorizeButton: Locator;
+  endpoints: Locator;
+  schemeContainer: Locator;
+  endpointBody: Locator;
+  tryItOutButton: Locator;
+  executeButton: Locator;
+  responseSection: Locator;
+  requestBody: Locator;
+};
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)